stencil.config.ts

```diff
@@ -10,30 +10,28 @@ export const config: Config = {
   globalStyle: "styles/main.scss",
   // globalScript: 'src/global/app.ts',
   taskQueue: "async",
-  outputTargets: [
-    {
-      type: "www",
-      serviceWorker: null,
-      baseUrl: "https://polls.pizza/",
-      dir: "dist/www",
-      prerenderConfig: "./prerender.config.ts",
-      copy: [{ src: "../public", dest: "." }],
-    },
-  ],
-  buildEs5: "prod",
   plugins: [
     replace({
       exclude: "node_modules/**",
       values: {
         "process.env.STRIPE_PUBLIC_KEY": process.env.STRIPE_PUBLIC_KEY ? `"${process.env.STRIPE_PUBLIC_KEY}"` : "process.env.STRIPE_PUBLIC_KEY",
-        "process.env.PIZZA_BASE_DOMAIN": process.env.PIZZA_BASE_DOMAIN ? `"${process.env.PIZZA_BASE_DOMAIN}"` : `"https://base.polls.pizza"`,
+        "process.env.PIZZA_BASE_DOMAIN": process.env.PIZZA_BASE_DOMAIN ? `"${process.env.PIZZA_BASE_DOMAIN}"` : `"z"`,
```

Comment:
want to leave this as `https://base-next.polls.pizza` so easier local dev?

Reply:
wow, no idea what even happened there

Reply:
I bet it was a typo ctrl+z -.-